src/mappings/UController/datasource.yaml

```diff
@@ -5,6 +5,7 @@ abis:
   - DAOToken
   - SchemeRegistrar
   - ContributionReward
+  - ContributionRewardExt
   - GenericScheme
```

Comment:
I think this is necessary if we want to make sure organizations which use UController can also use the ContributionRewardExt scheme. I think it's good to have it there as I fear not having it might cause issues with the subgraph if a dao we index adds this scheme.